packages/tradershub/src/components/Modal/Modal.tsx

```diff
@@ -34,7 +34,7 @@ type TModal = {
  * @property {string} [className] - Optional CSS class name
  */
 export type TModalComponents = {
-    children: ReactNode;
+    children?: ReactNode;
```

Comment:
What is the reason for this?

Reply:
@aizad-deriv the <Modal.Header> does not have children and just a title so it should be optinal